migrations/20240926071113_init.sql

```diff
@@ -0,0 +1,44 @@
+-- Users table to store user information
+CREATE TABLE users (
+    id BIGSERIAL PRIMARY KEY,
+    username VARCHAR(255) UNIQUE NOT NULL,
+    email VARCHAR(255) UNIQUE NOT NULL,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+-- Collections table
+CREATE TABLE collections (
+    id BIGSERIAL PRIMARY KEY,
+    parent_id BIGINT NULL REFERENCES collections(id),
+    user_id BIGINT REFERENCES users(id) ON DELETE CASCADE,
+    name VARCHAR(255) NOT NULL,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+-- Bookmarks table
+CREATE TABLE bookmarks (
+    id BIGSERIAL PRIMARY KEY,
+    collection_id BIGINT REFERENCES collections(id) ON DELETE CASCADE,
+    user_id BIGINT REFERENCES users(id) ON DELETE CASCADE,
+    url TEXT NOT NULL,
+    title VARCHAR(255) NOT NULL,
```

Comment:
change all varchar's to text

Reply:
Done. Also changed `TIMESTAMP` to `TIMESTAMPTZ` and added some tweaks to the docker compose.